app/lib/dttp/params/placement_assignment.rb

```diff
@@ -43,7 +43,7 @@ def build_params
           "dfe_courselevel" => COURSE_LEVEL_PG, # TODO: this can be PG (12) or UG (20).  Postgrad or undergrad. Hardcoded for now.
           "dfe_sendforregistration" => true,
           "dfe_ProviderId@odata.bind" => "/accounts(#{trainee.provider.dttp_id})",
-          "dfe_ITTQualificationAimId@odata.bind" => "/dfe_ittqualificationaims(#{ITT_QUALIFICATION_AIM_QTS})",
+          "dfe_ITTQualificationAimId@odata.bind" => "/dfe_ittqualificationaims(#{placement_assignment_qualification_aim_dttp_id})",
```

Comment:
Not a big deal, but I would use `dttp_qualification_aim_id(trainee.training_route)` directly instead of creating wrapper method.

Reply:
done